Chummer/Backend/Uniques/Contact.cs

```diff
@@ -893,6 +900,15 @@ public string Notes
             }
         }
 
+        /// <summary>
+        /// Forecolor to use for Notes in treeviews.
+        /// </summary>
+        public Color NotesColor
+        {
+            get => _colNotes;
+            set => _colNotes = value;
+        }
```

Comment:
These are not working, but at least not breaking anything that i tested.
As they implement IHasNotes, I think I should at least keep the changes in even if it might be a bit confusing if there are no effects when choosing a color.